app/views/layouts/application.html.erb

```diff
@@ -1,17 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>CibsSyllabusOrganizer</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>GoogleOauthDevise</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <style>
+        .flash__alert   { color: red; }
+        .flash__notice  { color: blue; }
+        .flash__success { color: green; }
+    </style>
   </head>
 
   <body>
-    <%= render partial: 'shared/navbar' %>
```

Comment:
Can you just change it so the login page doesn't render the navbar instead of removing it from all of the other pages?

Reply:
maybe works?